docs-examples/example-groovy/src/test/groovy/io/micronaut/rabbitmq/docs/consumer/concurrent/ProductListener.groovy

```diff
@@ -15,7 +15,7 @@ class ProductListener {
 
     CopyOnWriteArraySet<String> threads = new CopyOnWriteArraySet<>()
 
-    @Queue(value = "product", numberOfConsumers = 5) // <1>
+    @Queue(value = "product", numberOfConcurrentConsumers = "5") // <1>
```

Comment:
what is the motivation for changing the member name?

Reply:
I was not 100% sure if that is the proper approach, but I added a new member(`numberOfConcurrentConsumers`) and deprecated the old one(`numberOfConsumers`) to avoid breaking the backward compatibility contract.